### PR TITLE
chore: show the default home directory in `--help`

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/orchestrator-relayer/issues/270

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: cmd/qgb/orchestrator/config.go [hunk 1]
```diff
@@ -102,7 +106,11 @@ func parseOrchestratorFlags(cmd *cobra.Command) (StartConfig, error) {
 }
 
 func addInitFlags(cmd *cobra.Command) *cobra.Command {
-	cmd.Flags().String(base.FlagHome, "", "The qgb orchestrator home directory")
+	homeDir, err := base.DefaultServicePath("orchestrator")
```
Comment (evan-forbes) [inline cmd/qgb/orchestrator/config.go:109]: [not blocking]
we're using "orchestrator" twice here, so we might want to consider using a constant.

File: cmd/qgb/relayer/config.go [hunk 2]
```diff
@@ -38,7 +38,11 @@ func addRelayerStartFlags(cmd *cobra.Command) *cobra.Command {
 	cmd.Flags().StringP(FlagBootstrappers, "b", "", "Comma-separated multiaddresses of p2p peers to connect to")
 	cmd.Flags().StringP(FlagP2PNickname, "p", "", "Nickname of the p2p private key to use (if not provided, an existing one from the p2p store or a newly generated one will be used)")
 	cmd.Flags().StringP(FlagP2PListenAddress, "q", "/ip4/127.0.0.1/tcp/30000", "MultiAddr for the p2p peer to listen on")
-	cmd.Flags().String(base.FlagHome, "", "The qgb relayer home directory")
+	homeDir, err := base.DefaultServicePath("relayer")
```
Comment (evan-forbes) [inline cmd/qgb/relayer/config.go:41]: same as other comment

**Review by evan-forbes — APPROVED** [review]
only non-blocking comments

File: cmd/qgb/orchestrator/config.go (hunk 1 above)
Comment (SweeXordious) [reply cmd/qgb/orchestrator/config.go:109]: https://github.com/celestiaorg/orchestrator-relayer/issues/320

**Pull request merged**